internal/daysteps/daysteps.go

```diff
@@ -1,13 +1,64 @@
 package daysteps
 
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/Yandex-Practicum/tracker/internal/personaldata"
+	"github.com/Yandex-Practicum/tracker/internal/spentenergy"
+)
+
 type DaySteps struct {
-	// TODO: добавить поля
+	Steps    int
+	Duration time.Duration
+	personaldata.Personal
 }
 
 func (ds *DaySteps) Parse(datastring string) (err error) {
-	// TODO: реализовать функцию
+	parts := strings.Split(datastring, ",")
+	if len(parts) != 2 {
+		return fmt.Errorf("неверный формат данных: ожидается 2 части, получено %d", len(parts))
+	}
+
+	if strings.Contains(parts[0], " ") || strings.Contains(parts[1], " ") {
+		return fmt.Errorf("неверный формат данных: пробелы не допускаются")
+	}
+
+	steps, err := strconv.Atoi(parts[0])
+	if err != nil {
+		return fmt.Errorf("ошибка парсинга шагов: %v", err)
```

Comment:
При оборачивании нужно использовать `%w` (wrap), а просто при выводе в консоль `%v`
```suggestion
		return fmt.Errorf("ошибка парсинга шагов: %w", err)
```